image_classification/mobilenet_nchw.js

```diff
@@ -4,19 +4,20 @@ import {buildConstantByNpy, weightsOrigin} from '../common/utils.js';
 
 // MobileNet V2 model with 'nchw' input layout
 export class MobileNetV2Nchw {
-  constructor(dataType = 'float32') {
+  constructor(targetDataType = 'float32') {
     this.context_ = null;
     this.deviceType_ = null;
     this.builder_ = null;
     this.graph_ = null;
+    this.targetDataType_ = targetDataType;
```

Comment:
Ditto, you don't need to use this  `this.targetDataType_`, your previous code should work.